sound/soc/sof/ipc.c

```diff
@@ -534,7 +534,7 @@ static void ipc_period_elapsed(struct snd_sof_dev *sdev, u32 msg_id)
 		posn.host_posn, posn.dai_posn, posn.wallclock);
 
 	memcpy(&spcm->stream[direction].posn, &posn, sizeof(posn));
-
+	sbe_update_guest_posn(sdev, &posn);
```

Comment:
do we always need to do this ? - we have three operating modes, VBE, VFE and standalone. sure we dont need to always do this ?

Reply:
This function is also defined to update the position only in vBE. It will do nothing in standalone and vFE.

Reply:
ok, change comment to "optionally update position for vBE". Also func prefix should be sof_vbe_

Reply:
OK